### ISSUE TITLE
Sprite Font Word Wrap

### ISSUE THREAD

**Issue by 8bitThirsty** [Scirra/Construct-3-bugs#812]
## Problem description
Sprite font will not word wrap.  Using the default font.  Attempted switching between word and character and it would still not wrap.

## Attach a .c3p

[Bitmen.zip](https://github.com/Scirra/Construct-3-bugs/files/1309121/Bitmen.zip)

## Affected browsers

* Chrome: yes
Version 61.0.3163.91 (Official Build) (64-bit)

**Comment by AshleyScirra** [comment]
The project has 25 layouts and over 1000 events. Please see the bug report guidelines.

**Comment by 8bitThirsty** [comment]
You have the absolute worst customer service.  I have been a paying customer for a long time and you can't even bother to look into the bug reporting I am submitting because of some arbitrary rule.

**Comment by AshleyScirra** [comment]
The rule exists to ensure we can actually fix the problem you report. Very, very often the problem is just a mistake in events, so this is a necessary step to ensure there really is a problem, and that we can fix it. We have always had this rule, including throughout the lifetime of Construct 2, and we apply it to everyone.

**Comment by 8bitThirsty** [comment]
What I'm reading is you don't care if my large project works.  It's a simple function and you can't take 10 seconds to look into the issue because my project is large?  Why would I continue development in your system if it's going to continue to be like this?  I have put in so much time and while I'm paying you to also test your program so I will probably see this game through since I'm trapped but I won't forget this terrible customer service when I'm done.

**Comment by AshleyScirra** [comment]
We are keen to ensure Construct is stable, but it is simply infeasible to take a project with thousands of events and find yet again - as is very, very often the case - it's a mistake in the events, or a misunderstood feature. I'm afraid this is not something we have the resources to do. We provide clear guidelines on how to isolate problems to prove there is a bug and make a useful report that allows us to quickly and effectively fix the problem you are reporting. All our users abide by these rules and many of them are happy to. I've attached a project that demonstrates spritefont wrapping working correctly, which makes me strongly suspect the problem is really in your project and that this is not actually a bug.
[spritefont-wrap.zip](https://github.com/Scirra/Construct-3-bugs/files/1310533/spritefont-wrap.zip)

**Comment by 8bitThirsty** [comment]
If I create a brand new layout and event sheet with a brand new sprite font within my project and it still occurs will you look into?  

That's basically what I have already done in my project but I will make it more clear and list detailed steps to look into it.  My concern (which I have raised multiple times in bug reports I have filed) is that there is something wrong with my project file since I seem to run into so many of these one off issues and most of them are application crashes.  I have worked in C2 for years and C3 since the beginning and dealing with so many issues and the responses I have received are incredibly frustrating even when I have very solid evidence something messed up at no fault of my own.
